src/client/src/containers/AzureLogin/index.tsx

```diff
@@ -42,11 +46,38 @@ class AzureLogin extends React.Component<Props> {
       track: true
     });
   };
+  signOutClick = () => {
+    this.props.vscode.postMessage({
+      module: EXTENSION_MODULES.AZURE,
+      command: EXTENSION_COMMANDS.AZURE_LOGOUT,
+      track: true
+    });
+  };
+  keyDownClick = (event: any) => {
```

Comment:
keydown or click event? You can type these events using https://stackoverflow.com/questions/38337193/import-event-types-from-react-typescript

the nav looks pretty nice!

Reply:
thanks for the tip! Just added it